CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,70 @@
+# Tinted Theming Code of Conduct
+
+## Our Commitment
+
+Tinted Theming is committed to providing a welcoming and inclusive community where everyone feels respected and valued. We expect all community members, including contributors, maintainers, and users, to adhere to this Code of Conduct.
+
+## Inclusivity and Respect
+
+We value diversity and believe that a wide range of perspectives and experiences enriches our community. We are committed to creating an environment free from discrimination, harassment, and disrespectful behavior.
+
+## Language and Communication
+
+We strive for a community where everyone communicates with respect and empathy. Language has power, and we must use it responsibly.
+
+### Understanding Slurs and Offensive Language
+
+It's important to be aware that certain words, even those that might seem commonplace, can be harmful. This includes slurs or derogatory language directed at any disadvanged and underrepresented group, whether based on race, ethnicity, gender, disability, sexual orientation, or other characteristics. Such language:
+
+- Contributes to a culture of exclusion.
+- Creates an unwelcoming environment for affected individuals and groups.
+- Contradicts our values of respect and inclusivity.
+
+## Expected Behavior
+
+We expect all members of the Tinted Theming community to:
+
+1. Be respectful and considerate of others, regardless of their background, identity, or beliefs.
+2. Use inclusive language and avoid derogatory or offensive comments. This includes refraining from any slurs or derogatory language. If you're unsure whether a word might be offensive, we encourage erring on the side of caution and choosing more neutral terms.
+3. Listen actively and be open to different viewpoints and ideas.
+4. Provide constructive feedback and criticism in a respectful manner.
+5. Be patient and supportive, especially with newcomers.
+6. Respect and protect the privacy of others.
+7. Respect others' pronouns and gender identities. Avoid making assumptions about anyone's gender identity. Use gender-neutral language or the pronouns that individuals have specified for themselves.
```

Comment:
This is certainly something I'm less familiar with, so please let me know if I'm wrong.

Would this be clearer if we said to use gender neutral terms when unclear? Is specifically asking for gender neutral communication stricter than necessary?

In general, I try to use peoples' requested pronouns but if I don't know (or don't see them listed anywhere), I'll sometimes make a best guess, but be open to being corrected.

I'm open to the existing language, but I wanted to clarify before we finalize it.

Reply:
I've adjusted to wording to "encouraging" people to use gender neutral pronouns when they are unknown.